hedera-node/src/main/java/com/hedera/services/utils/accessors/AccessorFactory.java

```diff
@@ -62,10 +71,13 @@ public TxnAccessor triggeredTxn(
      * @param signedTxnWrapperBytes
      * @return
      */
-    private SignedTxnAccessor constructSpecializedAccessor(byte[] signedTxnWrapperBytes)
+    public TxnAccessor constructSpecializedAccessor(byte[] signedTxnWrapperBytes)
             throws InvalidProtocolBufferException {
-        // custom accessors will be defined here in future PR based on the function from
-        // functionExtractor
+        final var body = extractTransactionBody(Transaction.parseFrom(signedTxnWrapperBytes));
```

Comment:
This is good for now, but we'll definitely want to avoid parsing `signedTxnWrapperBytes` twice; i.e. the `TokenWipeAccessor` constructor and `SignedTxnAccessor.from()` factory should now probably take a `Transaction`.

Reply:
Makes sense. Will make that change

Reply:
Fixed !